src/main/java/org/breedinginsight/services/ProgramLocationService.java

```diff
@@ -208,7 +208,7 @@ public ProgramLocation create(AuthenticatedUser actingUser,
                 .accessibilityId(accessibilityId)
                 .topographyId(topographyId)
                 .abbreviation(programLocationRequest.getAbbreviation())
-                .coordinates(JSONB.valueOf(coordinates))
+                .coordinates(coordinates != null ? JSONB.valueOf(coordinates) : null)
```

Comment:
We should also do this for update as well right?

Reply:
Ah yep, thanks for catching that! Changed and pushed. Test updated.